src/index.js

```diff
@@ -285,6 +308,42 @@ class Searchbase {
         stateChanges: false
       });
     }
+    //  mic fields
+    this._micStatus = micStatus || MIC_STATUS.inactive;
+    this.micLanguage = micLanguage || 'en-US';
+    this._micResults = [];
+    this._micInstance =
+      window.webkitSpeechRecognition || window.SpeechRecognition || null;
+    if (!this._micInstance) {
+      console.error(
+        'SpeechRecognition is not supported in this browser. Please check the browser compatibility at https://developer.mozilla.org/en-US/docs/Web/API/SpeechRecognition#Browser_compatibility.'
+      );
+    }
+  }
```

Comment:
Don't create the instance in constructor.
Only create it when click event happens.